package.json

```diff
@@ -5,17 +5,19 @@
   "main": "readable.js",
   "dependencies": {
     "core-util-is": "~1.0.0",
-    "process-nextick-args": "~1.0.0",
     "inherits": "~2.0.1",
     "isarray": "0.0.1",
+    "process-nextick-args": "~1.0.0",
     "string_decoder": "~0.10.x",
     "util-deprecate": "~1.0.1"
   },
   "devDependencies": {
-    "tap": "~0.2.6"
+    "tap": "~0.2.6",
```

Comment:
tap is what is used to run the current tests from node, it's somewhat idiosyncratic how it runs at it looks like it just emits tap output via console logs so thats how it's interpreted, I suppose we could just run all the scrypts and feed it to tspec or something